app-modules/campaign/tests/Actions/TaskCampaignTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use App\Models\User;
+use Assist\Task\Models\Task;
+use Assist\Campaign\Models\Campaign;
+use Assist\Prospect\Models\Prospect;
+use Assist\Campaign\Models\CampaignAction;
+use Assist\Campaign\Enums\CampaignActionType;
+use Assist\CaseloadManagement\Models\Caseload;
+use Assist\CaseloadManagement\Enums\CaseloadType;
+
+it('will create the task records for educatables in the caseload', function () {
```

Comment:
If you could take a look at the test I wrote for my new campaign action recently. Let's add datasets so that this is run both on Students and Prospects. That way, we make sure this is working regardless of the Educatable of the caseload.